mobile/evidence/ios/Features/Sources/Profile/StatusViewModel.swift

```diff
@@ -59,3 +71,8 @@ public class StatusViewModel: ObservableObject {
     
 }
 
+extension StatusViewModel {
+    public func publicFuncIsAlertShowing() {
+        return self.isAlertShowing()
+    }
```

Comment:
This does not look like a good practice. If you need the func to be public, make it public.